core/src/main/java/pt/webdetails/cda/xml/DomVisitor.java

```diff
@@ -176,7 +176,7 @@ public void visit( SimpleDataAccess da, Element daEle ) {
 
     daEle.addAttribute( "id", da.getId() );
     daEle.addAttribute( "connection", da.getConnectionId() );
-    daEle.addAttribute( "type", da.getType() );
+    daEle.addAttribute( "type", da.getQueryType() );
```

Comment:
Won't this break things for other datasources?
And existing cda files don't need to be converted?

Reply:
da.queryType is got from element's "type" attribute
da.getType is used for displaying name of DataAccess
for all DataAccess types except DataservicesDataAccess those values are equal
so tests, which are running DomVisitor.visit( SimpleDataAccess da, Element daEle ) won't fail